Sources/IBLinterKit/Rules/CustomModuleRule.swift

```diff
@@ -70,41 +70,31 @@ extension Rules {
-                + viewControllers.flatMap { validate(for: $0, file: storyboard, fileNameWithoutExtension: storyboard.fileNameWithoutExtension) }
+                + viewControllers.flatMap { validate(forClassableObject: $0, file: storyboard, fileNameWithoutExtension: storyboard.fileNameWithoutExtension) }
         }
 
         private func validate<T: InterfaceBuilderFile>(for view: ViewProtocol, file: T, fileNameWithoutExtension: String) -> [Violation] {
-            let violation: [Violation] = {
-                guard let customClass = view.customClass else { return [] }
-                guard let expectedModule = moduleClasses.first(where: { $0.value.contains(customClass) }) else {
-                    return []
-                }
-                let message = "It does not match custom module rule in \(fileNameWithoutExtension). Custom module of \(customClass) is \(expectedModule.key)"
-                let violation = Violation(pathString: file.pathString, message: message, level: .error)
-                guard let customModule = view.customModule, expectedModule.key == customModule else {
-                    return [violation]
-                }
-                return []
-            }()
+            let violation = validate(forClassableObject: view, file: file, fileNameWithoutExtension: fileNameWithoutExtension)
             return violation + (view.subviews?.flatMap { validate(for: $0.view, file: file, fileNameWithoutExtension: fileNameWithoutExtension) } ?? [])
         }
 
-        private func validate<T: InterfaceBuilderFile>(for classableObject: IBCustomClassable, file: T, fileNameWithoutExtension: String) -> [Violation] {
+        private func validate<T: InterfaceBuilderFile>(
+            forClassableObject classableObject: IBCustomClassable, file: T, fileNameWithoutExtension: String
+        ) -> [Violation] {
             guard let customClass = classableObject.customClass else { return [] }
-            guard let expectedModule = moduleClasses.first(where: { $0.value.contains(customClass) }) else {
-                return []
-            }
-            let message = "It does not match custom module rule in \(fileNameWithoutExtension). Custom module of \(customClass) is \(expectedModule.key)"
-            let violation = Violation(pathString: file.pathString, message: message, level: .error)
-            guard let customModule = classableObject.customModule, expectedModule.key == customModule else {
+            let moduleCandidates = moduleClasses.lazy.filter({ $0.value.contains(customClass) }).map { $0.key }
+            guard !moduleCandidates.isEmpty else { return [] }
+            guard let customModule = classableObject.customModule, moduleCandidates.contains(customModule) else {
+                let message = "It does not match custom module rule in \(fileNameWithoutExtension). Custom module of \(customClass) is one of \(moduleCandidates)"
```

Comment:
```swift
\(moduleCandidates)
```

this outputs like this

```
LazyMapSequence<LazyFilterSequence<Dictionary<String, Array<String>>>, String>(_base: Swift.LazyFilterSequence<Swift.Dictionary<Swift.String, Swift.Array<Swift.String>>>(_base: ["ModuleA": ["AView", "BView", "CView",..... (a lot of classes)
```

I expects module name lists.

Reply:
I updated the message format.